docs/installation/settings.rst

```diff
@@ -0,0 +1,7 @@
+***************
+Tethys Settings
```

Comment:
I think we have several settings that are Tethys specific (e.g. TETHYS_DATABASES), and also several that come from Django plugins. I think it would be helpful to list all of the settings in the docs with a brief description and have links to the Django/plugin documentation where it applies. 

As for the conflict with app settings, this is a good point. Maybe we should call this file platform_settings.py? I'm not sure how best to avoid the confusion. Maybe we just need to explain it clearly in the docs.